api/ccapi/include/tensor_dim.h

```diff
@@ -65,15 +121,41 @@ class TensorDim {
    * formats of {w}, {h, w}, {c, h, w}, {b, c, h, w} for the NCHW & NHWC are
    * accepted
    */
-  TensorDim(std::initializer_list<size_t> dims, Format fm = Format::NCHW);
+  TensorDim(std::initializer_list<size_t> dims,
+            TensorType t_type_ = TensorType());
+
+  // TensorDim(std::initializer_list<size_t> dims, TensorDim::Format
+  // fm=Format::NCHW,
+  //           TensorDim::DataType d_type=DataType::FP32);
 
   /**
    * @brief Construct a new Tensor Dim object without batch dimension
    *
    * @param shapes shapes without batch dimension
    * @param fm format NCHW | HNWC
    */
-  TensorDim(const std::array<size_t, 3> &shapes, Format fm = Format::NCHW);
+  TensorDim(const std::array<size_t, 3> &shapes,
+            TensorType t_type_ = TensorType());
+
+  // TensorDim(const std::array<size_t, 3> &shapes, TensorDim::Format fm =
```

Comment:
Should we remove the commented code OR add a revisit comment?

Reply:
I will remove. thanks.